libr/main/rabin2.c

```diff
@@ -1151,6 +1189,7 @@ R_API int r_main_rabin2(int argc, char **argv) {
 	run_action ("versioninfo", R_BIN_REQ_VERSIONINFO, R_CORE_BIN_ACC_VERSIONINFO);
 	run_action ("sections", R_BIN_REQ_SIGNATURE, R_CORE_BIN_ACC_SIGNATURE);
 	run_action ("hashes", R_BIN_REQ_HASHES, R_CORE_BIN_ACC_HASHES);
+	run_action ("sections mapping", R_BIN_REQ_SECTIONS_MAPPING, R_CORE_BIN_ACC_SECTIONS_MAPPING);
```

Comment:
I think it's useful to add `R_BIN_REQ_SECTIONS_MAPPING` at https://github.com/radareorg/radare2/blob/master/libr/main/rabin2.c#L652 as well. Also, please update the help message at https://github.com/radareorg/radare2/blob/master/libr/main/rabin2.c#L32